app/views/manage/users/show.html.haml

```diff
@@ -2,4 +2,30 @@
 
 %h1= @user.name
 %h2= @user.email
-  
\ No newline at end of file
+
+%h3 Your Indices:
+%table.table.table-condensed.table-striped
+  %thead
+    %tr
+      %th.col-sm-3 Community
+      %th.col-sm-2 AWRVI Index
+      %th.col-sm-2 Last Updated
+      %th.col-sm-3 Progress
+      %th.col-sm-2 View AWRVI Dashboard
+  %tbody
+    - @indices.each do |index|
+      %tr
+        %td
+          %b= index.community.name
+        %td
+          #{number_with_precision(index.vulnerability_index, precision: 2)}
+          - unless index.published_at.nil?
+            .label.label-primary Published
+        %td
+          #{time_ago_in_words index.updated_at} ago
+        %td
+          .progress
+            - percentage = index.completeness_percentage
+            .progress-bar{style: "width: #{percentage}%", class: progress_bar_for_percentage(percentage)}
```

Comment:
Do we want to use `progress_bar_for_percentage` for the completeness?  That's really intended for the vulnerability index coloring handling